lib/view_model/transaction_details_view_model.dart

```diff
@@ -214,7 +214,7 @@ abstract class TransactionDetailsViewModelBase with Store {
     final addressIndex = tx.additionalInfo['addressIndex'] as int;
     final feeFormatted = tx.feeFormatted();
     final _items = [
-      StandartListItem(title: S.current.transaction_details_transaction_id, value: tx.id),
+      StandartListItem(title: S.current.transaction_details_transaction_id, value: tx.txhash),
```

Comment:
this should be changed then in other places in this file
for 2 reasons.
1. there are some places that uses `.id` regardless of the wallet type
2. to maintain consistency